src/internal/m365/collection/drive/collection.go

```diff
@@ -589,14 +589,6 @@ func (oc *Collection) streamDriveItem(
 		parentPath)
 
 	ctx = clues.Add(ctx, "item_info", itemInfo)
-	// Drive content download requests are also rate limited by graph api.
-	// Ensure that this request goes through the drive limiter & not the default
-	// limiter.
-	ctx = graph.BindRateLimiterConfig(
```

Comment:
We don't need to specify this anymore, since the ctx bindings will already exist when we enter this function.